release-notes/5.0/api-diff/netstandard2.1/5.0_System.Globalization.md

```diff
@@ -29,6 +29,7 @@
 +        public SortKey GetSortKey(string source, CompareOptions options);
 +        public int GetSortKeyLength(ReadOnlySpan<char> source, CompareOptions options = CompareOptions.None);
 +        public int IndexOf(ReadOnlySpan<char> source, ReadOnlySpan<char> value, CompareOptions options = CompareOptions.None);
++        public int IndexOf(ReadOnlySpan<char> source, ReadOnlySpan<char> value, CompareOptions options, out int matchLength);
```

Comment:
@GrabYourPitchforks is there any reason why the diff here not included in ns2,1?